platform/android/export/export.cpp

```diff
@@ -1443,23 +1443,87 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		//printf("end\n");
 	}
 
-	void _process_launcher_icons(const String &p_processing_file_name, const Ref<Image> &p_source_image, const LauncherIcon p_icon, Vector<uint8_t> &p_data) {
-		if (p_processing_file_name == p_icon.export_path) {
-			Ref<Image> working_image = p_source_image;
+	void _process_launcher_icons(const String &p_file_name, Vector<uint8_t> &p_data, const Ref<Image> &p_source_image, int dimension) {
```

Comment:
No, I think it's because originally I had that other method `resize_launcher_icons` and the method signatures might have gotten messed up there. I can change it back if it's important.